### ISSUE TITLE
Feature request - Auto password checker & export function

### ISSUE THREAD

**Issue by Lexus89** [routerkeygen/routerkeygenPC#36]
For some AP's many possible keys are generated and it would be nice to have an option to actually test all of them to see if one works. Also having an export function for all possible keys would be handy.

**Comment by DoctorD90** [comment]
An export function as in apk, to export all keys to a plain text file would be nice!

**Comment by RealEnder** [comment]
Something like this?

alex@osi:~/c/custom_crack$ ./routerkeygen -q -s Alice-46440251 -m 00:23:8e:5b:eb:2c > a.txt
alex@osi:~/c/custom_crack$ cat a.txt
infpjvms3jidycdm4i0fsxvl
0y64yxubezWeh29kkjm7s6j1

**Comment by DoctorD90** [comment]
exactly!!! this would be nice to get in GUI or basically also from command line :) so that even with data from other software we can automate generation of keys (when available) :+1:

**Comment by RealEnder** [comment]
I plan to implement RESTful API to routerkeygenPC for integration, but no time for now...

**Comment by DoctorD90** [comment]
Thx! I checked your command after your edit, so I gotcha that from command line is possible yet. This is yet a great stuff! Thx!

**Comment by yarda** [comment]
I proposed addition of the "copy all" button to the main UI in #39
